tests/unit/models/membership-test.js

```diff
@@ -109,7 +109,7 @@ module('Unit | Model | membership', function (hooks) {
             role: role,
           });
 
-          organization.memberships.pushObject(model);
+          organization.memberships.push(model);
```

Comment:
Some of these `.push` calls seem to fail after updating to v5, but the tests passed on v4. I need to double check that.